gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -2,360 +2,12 @@
 -- @class HELPSCRN
 -- @desc Help screen
 
-local GetTranslation = LANG.GetTranslation
-local GetPTranslation = LANG.GetParamTranslation
-local pairs = pairs
-local ipairs = ipairs
 local IsValid = IsValid
-local ConVarExists = ConVarExists
 local CreateConVar = CreateConVar
-local surface = surface
-
-ttt_include("vgui__cl_f1settings_button")
-
-surface.CreateFont("SettingsButtonFont", {font = "Trebuchet24", size = 24, weight = 1000})
 
 CreateConVar("ttt_spectator_mode", "0", FCVAR_ARCHIVE)
 CreateConVar("ttt_mute_team_check", "0")
```

Comment:
These ConVars +corresponding cvar callbacks should moved out of here.

Reply:
probably. But they were in there for ever and I did not know where to put them. Any idea?

Reply:
Mmh difficult, maybe at least rather to the gameplay help populate file.